### ISSUE TITLE
Return isodelay time in dzrf

### ISSUE THREAD

**Issue by ShannonZ** [mikgroup/sigpy#104]
ISODELAY time of a simulated RF pulse is needed to make it really useful.

**What's I want**
sigpy.mri.rf.slr.dzrf() return designed RF pulse and its isodelay.

**Comment by jonbmartin** [comment]
Agreed, this would be a useful thing for the function to deliver! In progress now.

**Comment by sidward** [comment]
Hi @jonbmartin,

Wanted to check in: Any update on this?

**Comment by jonbmartin** [comment]
In progress! The function itself is actually written (PR #112) but fails checks, as it will require the updates to meet python 3.8 standards in PR #113. Working on that PR now.

**Comment by ShannonZ** [comment]
@jonbmartin I cannot find any information about how to calculate the **isodelay of a shaped pulse** from the reference you given in the comments.
 _Bernstein, M.A. King, K.F. and Zhou, X.J. (2004). Handbook of MRI pulse sequences. Amsterdam: Academic Press._
 IMOO, for symmetric pulses the isodelay is typically half the RF duration, for SLR or optimized RF pulses the isodelay further depends on the flip angle. I'm not sure whether or not it's just the duration from the peak to the end.
 `np.size(rf) - np.argmax(abs(rf)))*dt`
